src/content/developers/docs/gas/index.md

```diff
@@ -23,86 +22,102 @@ _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/d
 
 In essence, gas fees are paid in Ethereum's native currency, ether (ETH). Gas prices are denoted in gwei, which itself is a denomination of ETH - each gwei is equal to 0.000000001 ETH (10<sup>-9</sup> ETH). For example, instead of saying that your gas costs 0.000000001 ether, you can say your gas costs 1 gwei.
 
-Let's say Alice has to pay Bob 1ETH.
+## Prior to the London update {#pre-london}
+
+The way transaction fees on the Ethereum network are calculated changed during the London update. Here is a recap of how things used to work:
+
+Let's say Alice had to pay Bob 1ETH.
 In the transaction the gas limit is 21,000 units and the gas price is 200 gwei.
 
-Total fee will be: `Gas units (limit) * Gas price per unit`
+Total fee would have been: `Gas units (limit) * Gas price per unit`
 i.e `21,000 * 200 = 4,200,000 gwei` or 0.0042 ETH
 
-Now, when Alice sends the money, 1.0042 ETH will be deducted from Alice's account.
-Bob will be credited 1.0000 ETH.
-Miner gets 0.0042 ETH.
+Now, when Alice sent the money, 1.0042 ETH will be deducted from Alice's account.
+Bob would be credited 1.0000 ETH.
+Miner got 0.0042 ETH.
 
 This video offers a concise overview of gas and why it exists:
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/AJvzNICwcwc" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Why do gas fees exist? {#why-do-gas-fees-exist}
+## After to the London update {#post-london}
 
-In short, gas fees help keep the Ethereum network secure. By requiring a fee for every computation executed on the network, we prevent actors from spamming the network. In order to prevent accidental or hostile infinite loops or other computational wastage in code, each transaction is required to set a limit to how many computational steps of code execution it can use. The fundamental unit of computation is "gas".
+Starting with [the London update](/history/#london), every block has a base fee, the minimum per gas price for inclusion in this block. Optionally, users can set a priority fee (tip).
```

Comment:
```suggestion
Starting with [the London network upgrade](/history/#london), every block has a base fee - the minimum per gas price for inclusion in this block. Optionally, users can set a priority fee (tip).
```

Reply:
"the minimum per gas price" sounds a bit weird. Is it accurate to say "minimum gas price"?

Reply:
I think it's important to explicitly call out that now there is 2 elements of transaction fees:  1) the base fee & 2) the tip. Previously there was only 1. Perhaps "base fees" & "tips" sections should be explicitly numbered as subheaders?

Reply:
> "the minimum per gas price" sounds a bit weird. Is it accurate to say "minimum gas price"?

I don't think so because ``baseFee * gasLimit = minimumGasPrice``. How about 'minimum price per unit of gas'?

Reply:
Ya I like that.